charts/localstack/values.yaml

```diff
@@ -149,3 +152,13 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# Mount /docker-entrypoint-initaws.d to run startup scripts with
+# {{ template "localstack.fullname" . }}-init-scripts-config configMap
+enableStartupScripts: true
```

Comment:
I would not default this to true - I don't think we want to change the upstream defaults for other users. The existing behavior is that this is false by default because the setting is missing entirely.